### ISSUE TITLE
Text missing from the bottom of scope and content on collection level

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/vt-arclight#488]
<img width="685" alt="Screen Shot 2023-01-19 at 1 34 52 PM" src="https://user-images.githubusercontent.com/3269689/213567320-dfae25dc-9c88-4b35-aa76-4d5a7ac3ef70.png">
 
Text stops only describing the first charge, not all three. Here is the complete text that should be there **AFTER the bulleted list of defendants:**

Article 6 of the [IMT’s Charter](https://treaties.un.org/pages/showDetails.aspx?objid=08000002801572a2) listed three broad categories of crimes:

(a) Crimes against peace: namely, planning, preparation, initiation or waging of a war of aggression, or a war in violation of international treaties, agreements or assurances, or participation in a common plan or conspiracy for the accomplishment of any of the foregoing;

(b) War crimes: namely, violations of the laws or customs of war. Such violations shall include, but not be limited to, murder, ill-treatment or deportation to slave labour or for any other purpose of civilian population of or in occupied territory, murder or ill-treatment of prisoners of war or persons on the seas, killing of hostages, plunder of public or private property, wanton destruction of cities, towns or villages, or devastation not justified by military necessity;

(c) Crimes against humanity: namely, murder, extermination, enslavement, deportation, and other inhumane acts committed against any civilian population, before or during the war, or persecutions on political, racial or religious grounds in execution of or in connection with any crime within the jurisdiction of the Tribunal, whether or not in violation of the domestic law of the country where perpetrated.

The first of these charges, (a), was subdivided in the indictment, which thus contained four counts, of which the defendants each faced his own permutation.

Count one (common plan or conspiracy) was prosecuted by the United States, and count two (crimes against peace) by the United Kingdom. Count three (war crimes) and count four (crimes against humanity) were prosecuted jointly by France and the Soviet Union, according to whether the crimes in question had been committed in Western or Eastern Europe.

**Comment by marlo-longley** [comment]
Hey @laurensorensen -- looks like 2 issues:

1. The text in ASpace prod does not match this text 
2. The text displayed in the application does not match what is in ASpace prod

**Comment by laurensorensen** [comment]
Ah, weird! Must have been a copy/paste error. What would be a next step? I can update the text in ASpace prod

**Comment by marlo-longley** [comment]
Sure let's do that, then I can debug why the whole text isn't displaying. I suspect bad HTML.

**Comment by laurensorensen** [comment]
Ok, I've updated this in ASpace, prod and stage. Unfortunately it looks like it is an ASpace error - e.g. I updated this yesterday and apparently it didn't save the rest of the text. See screenshot. I'm not sure what the next steps would be here, a meeting with Michael Olson? 
<img width="824" alt="Screen Shot 2023-01-20 at 12 40 23 PM" src="https://user-images.githubusercontent.com/3269689/213806968-09f447ae-bc7b-44a8-bd56-11b7ba0c51b1.png">
